NEWS.rst

```diff
@@ -4417,6 +4417,7 @@ Twisted Core 15.0.0 (2015-01-24)
 
 Features
 --------
+ - twisted.internet.endpoints adding support for cipher in `serverFromString`(#12134)
```

Comment:
This needs to be moved into a "fragment" file.

We don't do direct edit of the release notes.

See more info here https://docs.twisted.org/en/stable/development/dev-process.html#release-notes-management

Reply:
[#6e30f](https://github.com/twisted/twisted/pull/12137/commits/6e30f9f83d118633e7be1c489d00073549c1b285)